website/src/layout.tsx

```diff
@@ -9,13 +9,32 @@ import Footer from "./footer"
 import "./global-styles.css"
 import * as css from "./layout.css"
 import { MobileNav, NavMenu } from "./menu"
+import { HeaderPrefDrawer, selectedMode, selectedPhonetics } from "./mode"
 import { hideOnPrint, themeClass } from "./sprinkles.css"
+import { PhoneticRepresentation, ViewMode } from "./types"
 import "./wordpress.css"
 
 /* const ClientSignIn = lazy(() => import("./client/signin")) */
 
+// Set up experiencelevel
+export const preferencesContext = React.createContext({
```

Comment:
Move this into the types file or a new file and give it an upper camel case name. I.e. `PreferencesContext`. And then make a new component `PreferencesProvider` that wraps all the state management hooks you're doing in the Layout  omponent right now. Then, in Layout, you should only need to add `PreferencesProvider` wrapped around the return value.